src/game/common/audio/audiomanager.cpp

```diff
@@ -139,10 +141,66 @@ void AudioManager::Reset()
  */
 void AudioManager::Update()
 {
-    // TODO Requires g_theTacticalView.
-#ifdef GAME_DLL
-    Call_Method<void, AudioManager>(PICK_ADDRESS(0x00404FB0, 0x006E3ADF), this);
-#endif
+    Coord3D view_pos = g_theTacticalView->Get_Position();
+    float view_angle = g_theTacticalView->Get_Angle();
+    Matrix3D tm(Matrix3D::Identity);
+    tm.Rotate_Z(view_angle);
+    Vector3 v(0.0f, 1.0f, 0.0f);
+    v = tm * v;
```

Comment:
this used a multvector3, is that really the same?

Reply:
Yes, this is the same.